dataset/dictionary/characters-nodkrai.ts

```diff
@@ -311,15 +311,15 @@ export default [
     ja: "ルシーネ",
     zhCN: "鲁什涅",
     zhTW: "魯什涅",
-    notes: "ナシャタウンにいる人物。ライトキーパー",
+    notes: "世界任務「哀しき鈴の音」に登場する、ナシャタウンにいる人物。ライトキーパー",
     tags: [ "nodkrai", "character-sub" ],
   },
   {
     en: "Sukhovykh",
     ja: "スホヴィーク",
     zhCN: "苏霍维赫",
     zhTW: "蘇霍維赫",
-    notes: "ナシャタウンにいる人物。ライトキーパー",
+    notes: "世界任務「哀しき鈴の音」に登場する、ナシャタウンにいる人物。ライトキーパー",
```

Comment:
I guess it is a bit unnatural Japanese. How about rewriting like this?

```suggestion
    notes: "ナシャタウンにいる人物。世界任務「哀しき鈴の音」に登場するライトキーパー",
```